src/main/java/app/controllers/UserController.java

```diff
@@ -70,13 +70,13 @@ public static void login(Context ctx, ConnectionPool connectionPool)
             ctx.sessionAttribute("currentUser", user);
             // Hvis ja, send videre til forsiden med login besked
             ctx.attribute("message", "Du er nu logget ind");
-            ctx.render("index.html");
+            ctx.render("startPage.html");
```

Comment:
please keep using the index as it will otherwise break the application for all other teams